From 1cde7b66473d90d9735edf451a7b8359e1dfc6a4 Mon Sep 17 00:00:00 2001
From: Martijn Braam <martijn@brixit.nl>
Date: Fri, 4 Sep 2020 17:35:39 +0200
Subject: media: gc2145: Added BGGR bayer mode

Not all raw bayer modes from the sensor match up with the ones defined
in v4l, mostly because they're mirrored.
---
 drivers/media/i2c/gc2145.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/media/i2c/gc2145.c b/drivers/media/i2c/gc2145.c
index cc85ac772d5a..7539b122eeef 100644
--- a/drivers/media/i2c/gc2145.c
+++ b/drivers/media/i2c/gc2145.c
@@ -187,6 +187,11 @@ static const struct gc2145_pixfmt gc2145_formats[] = {
 		.colorspace        = V4L2_COLORSPACE_SRGB,
 		.fmt_setup         = 0x06,
 	},
+	{
+		.code              = MEDIA_BUS_FMT_SBGGR8_1X8,
+		.colorspace        = V4L2_COLORSPACE_RAW,
+		.fmt_setup         = 0x17,
+	},
 };
 
 static const struct gc2145_pixfmt *gc2145_find_format(u32 code)
-- 
2.35.3

